<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GleisnetzWindow</class>
 <widget class="QMainWindow" name="GleisnetzWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gleisnetz</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="signal_tab">
       <attribute name="title">
        <string>Gleisplan</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QScrollArea" name="scrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="signal_graph_area">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>68</width>
                 <height>515</height>
                </rect>
               </property>
              </widget>
             </widget>
             <widget class="QFrame" name="signal_settings_frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QCheckBox" name="signal_weichen_check">
                 <property name="text">
                  <string>Weichen entfernen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="signal_anschluss_check">
                 <property name="text">
                  <string>Anschlüsse prüfen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="signal_bahnsteig_check">
                 <property name="text">
                  <string>Ausfahrsignale entfernen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="signal_paar_check">
                 <property name="text">
                  <string>Blocksignale entfernen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="signal_zwischen_check">
                 <property name="text">
                  <string>Zwischensignale entfernen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="signal_schleifen_check">
                 <property name="text">
                  <string>Schleifen auflösen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="signal_nachbarn_check">
                 <property name="text">
                  <string>Nachbarbahnsteige vereinen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="signal_reserve1_check">
                 <property name="text">
                  <string>(keine Funktion)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="signal_aktualisieren_button">
                 <property name="text">
                  <string>Grafik aktualisieren</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>413</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="bahnhof_tab">
       <attribute name="title">
        <string>Bahnhofplan</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QScrollArea" name="scrollArea_2">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="bahnhof_graph_area">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>68</width>
                 <height>515</height>
                </rect>
               </property>
              </widget>
             </widget>
             <widget class="QFrame" name="bahnhof_settings_frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QCheckBox" name="bahnhof_leerverbindung_check">
                 <property name="text">
                  <string>Leere Verbindungen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="bahnhof_check_1">
                 <property name="text">
                  <string>Keine Funktion</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="bahnhof_check_2">
                 <property name="text">
                  <string>Keine Funktion</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="bahnhof_check_3">
                 <property name="text">
                  <string>Keine Funktion</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="bahnhof_aktualisieren_button">
                 <property name="text">
                  <string>Grafik aktualisieren</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>364</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="linien_tab">
       <attribute name="title">
        <string>Liniennetz</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QSplitter" name="splitter_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QScrollArea" name="scrollArea_3">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="linien_graph_area">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>68</width>
                 <height>515</height>
                </rect>
               </property>
              </widget>
             </widget>
             <widget class="QFrame" name="linien_settings_frame">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QCheckBox" name="linien_schleifen_check">
                 <property name="text">
                  <string>Schleifen auflösen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="linien_check_2">
                 <property name="text">
                  <string>Keine Funktion</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="linien_check_3">
                 <property name="text">
                  <string>Keine Funktion</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="linien_aktualisieren_button">
                 <property name="text">
                  <string>Grafik aktualisieren</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>384</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
